doc/tutorial.txt

```diff
@@ -66,7 +66,7 @@ $ tree -d
 
 the configure step becomes
 
-./configure --host=aarch64-linux-gnu LDFLAGS="--static -L./../json-c/." CFLAGS="-I./.." --with-deadline
+./configure --host=aarch64-linux-gnu LDFLAGS="-L$PWD/../json-c" CFLAGS="-I./../json-c" --with-deadline
```

Comment:
This looks now good to me and generates a statically linked binary.
However, with a directory tree like in the above example LDFLAGS is not correct, as it should be
```
./configure --host=aarch64-linux-gnu LDFLAGS="-L$PWD/json-c" CFLAGS="-I./../json-c" --with-deadline
```

Reply:
you're right, i have json-c in the parent directory (i don't like nesting unrelated projects' build directories). could either fix the tree or the invocation, whichever you prefer.

just for clarification, i think that should have been
```
./configure --host=aarch64-linux-gnu LDFLAGS="-L$PWD/json-c" CFLAGS="-I./json-c" --with-deadline
```
instead of what you wrote above..(note the CFLAGS)

Reply:
Let's go with yours. Please update the text accordingly.

Reply:
done that. is it ok?